transforms/.make.cicd.targets

```diff
@@ -28,62 +32,113 @@ clean:: .transforms.clean
 set-versions::
 
 ## We need to think how we want to do this going forward
-build::
 
-image::
-	@if [ -e Dockerfile ]; then \
-		$(MAKE) image-default ; \
-	else \
-		echo "Skipping image for $(shell pwd) since no Dockerfile is present"; \
+build:: image
+
+publish:
+	@if [ -e Dockerfile.python ]; then \
```

Comment:
maybe split this into publish-python/ray/spark.

Reply:
@daw3rd When is publish being used. Do you know ?